### DISCUSSION TITLE
Implement Matter in React Native

### DISCUSSION THREAD

**Discussion by joey01245** [matter-js/matter.js#1185] [Q&A]
Hi everyone,

Has anyone successfully used this package with React Native and `matter.js-react-native`? If so, could you share an example?

**Comment by Apollon77** [comment]
Hi,

no and most likely it will also not fully work. We are still in the process to get it to work and solve the issues that pop up one after the other.

But you are welcome to try it :-) UDP works, also initial connection to the device via UDP works, I expect he next issue somewhere in crypto area (again) ...

Ingo
